src/libs/CardUtils.ts

```diff
@@ -130,6 +130,21 @@ function maskCard(lastFour = ''): string {
     return maskedString.replace(/(.{4})/g, '$1 ').trim();
 }
 
+/**
+ * Returns a masked credit card string.
+ * Converts given 'X' to '•' for the entire card string.
+ *
+ * @param [cardName=""] - card name with XXXX in the middle.
```

Comment:
Its not optional anymore, I wont block on this but can you please address it in a future PR?
```suggestion
 * @param cardName - card name with XXXX in the middle.
```